components/server/ee/src/billing/billing-mode.ts

```diff
@@ -196,7 +196,15 @@ export class BillingModesImpl implements BillingModes {
             return { mode: "chargebee" };
         }
 
-        // 3. If not: we don't even have to check for a team subscription
-        return { mode: "usage-based" };
+        // 3. Now we're usage-based. We only have to figure out whether we have a plan yet or not.
+        const result: BillingMode = { mode: "usage-based" };
+        const customer = await this.stripeSvc.findCustomerByUserId(team.id);
+        if (customer) {
+            const subscription = await this.stripeSvc.findUncancelledSubscriptionByCustomer(customer.id);
+            if (subscription) {
+                result.paid = true;
```

Comment:
Ah, interesting, so we _do_ potentially set it for users as well (assuming they somehow get a Stripe subscription). This somewhat contradicts earlier comments, which say `.paid` is only for teams. 💭

Reply:
This is only set for teams, because that's where we need to distinguish atm. But can imagine that it also helps for users in the future, just want to avoid more complexity.

Reply:
Ah, maybe you wanted to have `findCustomerByTeamId` instead of `findCustomerByUserId` above? 👀

Reply:
:scream: Thank you! Will fix here, as well as in the tests.